modules/installation-about-restricted-network.adoc

```diff
@@ -53,6 +53,13 @@ endif::[]
 In {product-title} {product-version}, you can perform an installation that does not
 require an active connection to the internet to obtain software components. Restricted network installations can be completed using installer-provisioned infrastructure or user-provisioned infrastructure, depending on the cloud platform to which you are installing the cluster.
 
+ifdef::ipi[]
```

Comment:
It's just for powervs IPI, is there a way to limit?

Reply:
Yes, the conditions in the `ifdef`/`endif` construction need to isolate only PowerVS IPI in that case.

Reply:
@jeana-redhat fyi: I have removed the ifdef / endif condition.

request you to check. Thanks.

Reply:
Hey @alishaIBM, if this applies only to PowerVS IPI (as I understand from @mjturek above), then the condition must make sure that this only appears in the PowerVS IPI docs. 

It was previously showing in all the platforms I listed above. By removing the condition, it will now show in _all_ assemblies that use this module. It needs a condition that _only_ applies to PowerVS IPI.

Reply:
@alishaIBM add a new condition on line 28 of modules/installation-about-restricted-networks.adoc like this:
```
ifeval::["{context}" == "installing-restricted-networks-ibm-power-vs"]
:ipi:
:ipi-powervs:
endif::[]
```

Then re-add an ifdef on line 56 that contains the note, and a corresponding `endif` afterwards:
`ifdef::ipi-powervs[]`

Make sure to add the new condition to the end of the file where all the variables are unset, like:
```
ifeval::["{context}" == "installing-restricted-networks-ibm-power-vs"]
:!ipi:
:!ipi-powervs:
endif::[]
```

Reply:
@jeana-redhat @bscott-rh thanks for the clarification. I have updated the code as shown above. Request you to take a look. Thanks.

Reply:
Thanks @alishaIBM :tada: